src/lib/firebase.js

```diff
@@ -345,7 +345,7 @@ foam.CLASS({
   extends: 'com.firebase.FirebaseDAO',
```

Comment:
Ignore file: Feature required universal HTTPRequest. Will merge in #248 after it's reviewed.